test/compose/test-compose

```diff
@@ -164,20 +167,22 @@ function test_port() {
     local expect="$3"            # what to expect from curl output
 
     # -s -S means "silent, but show errors"
-    local actual=$(curl --retry 3 --retry-all-errors -s -S http://127.0.0.1:$port/)
+    local actual
+    actual=$(curl --retry 3 --retry-all-errors -s -S http://127.0.0.1:$port/)
     local curl_rc=$?
 
-    # FIXME 2021-04-13: test is flaking, curl succeeds but returns empty result.
-    # Could it be that the container is not actually ready? Wait, and retry.
-    if [[ $curl_rc -eq 0 && -z "$actual" ]]; then
-        sleep 1
-        echo "# Retrying curl:"
-        actual=$(curl --retry 3 --retry-all-errors -s -S http://127.0.0.1:$port/)
-        curl_rc=$?
-    fi
-
     if [ $curl_rc -ne 0 ]; then
-        _show_ok 0 "$testname - curl failed with status $curl_rc"
+        _show_ok 0 "$testname - curl (port $port) failed with status $curl_rc"
+        # FIXME: is this useful? What else can we do to diagnose?
+        echo "# docker-compose logs:"
+        docker-compose logs
+        echo "# podman ps -a:"
+        $PODMAN_BIN ps -a
```

Comment:
Sry I just noticed, this wont work because of the custom --root ... You have to use the podman function

Reply:
Oh, good catch, thanks. Re-pushed.